specs/core/0_beacon-chain.md

```diff
@@ -207,6 +207,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value | Unit | Duration |
 | - | - | :-: | :-: |
+| `SECONDS_PER_SLOT` | `6` | seconds | 6 seconds |
```

Comment:
`SECONDS_PER_SLOT` is not used by the state transition function

Reply:
@JustinDrake The reason is about spec file dependencies.
* `0_beacon_chain.md` is the kind of the "main spec file" that `0_fork-choice.md` imports.
* However, in the duration note: "`SLOTS_PER_EPOCH`: 64 slots = 6.4 minutes" implies that the fact `SECONDS_PER_SLOT` is `6`.

Do you think it makes sense?

Reply:
I agree. 6 seconds and a slot time are implied all over this spec and it seems incomplete to hide it in another file. The validator doc also uses it and it seems strange to have it inherit the var from fork choice.